Documentation/MixedRealityConfigurationGuide.md

```diff
@@ -22,13 +22,13 @@ The main configuration profile, which is attached to the *MixedRealityToolkit* G
 
```

Comment:
with the line <The Mixed Reality Toolkit "locks" ...> can you add in this note, that the MRTK configuration is not editable during playmode (see this PR #4991 ). A

Reply:
Updated